spec/Vehicle/Vehicle.vspec

```diff
@@ -214,7 +241,10 @@ AverageSpeed:
   type: sensor
   unit: km/h
   description: Average speed for the current trip.
-
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
```

Comment:
Is it sufficient to have a single `AverageSpeed` - is so do we want to keep old name, or rename to `AverageSpeedSinceStart` to be better aligned with other signals referring to current trip.

Or do we need additional signals for average speed - e.g. by adding one related to average speed over lifetime or average speed since trip meter reset?